src/general/StageHUDBase.cs

```diff
@@ -283,9 +287,10 @@ public abstract class StageHUDBase<TStage> : Control, IStageHUD
     private bool compoundCompressed;
 
     /// <summary>
-    ///   This is opposite of the expected value, ie. this is true when the panels are collapsed
+    ///   These are the opposite of the expected value, ie. these are true when the panels are collapsed
     /// </summary>
-    private bool leftPanelsActive;
+    private bool compoundPanelsActive;
```

Comment:
There's only one panel for all compounds right? So I guess this name should be `compoundsPanelActive`.